gherkin-frontend/src/main/java/org/sonar/gherkin/parser/GherkinGrammar.java

```diff
@@ -93,7 +93,7 @@ public ScenarioOutlineTree SCENARIO_OUTLINE() {
         b.optional(NAME()),
         b.optional(SCENARIO_DESCRIPTION()),
         b.zeroOrMore(STEP()),
-        EXAMPLES()));
+        b.oneOrMore(EXAMPLES())));
   }
 
   public ExamplesTree EXAMPLES() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)